independent-publisher-connectors/JiraSearch/apiDefinition.swagger.json

```diff
@@ -0,0 +1,392 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "JIRA Search",
+    "description": "Searches for JIRA issues using JQL expression using the GET pattern. More information can be found at: https://docs.atlassian.com/software/jira/docs/api/REST/8.16.2/#api/2/search-search",
+    "version": "1.0",
+    "contact" : {
+	  "name" : "Paul Culmsee",
+	  "url" : "https://www.atlassian.com/software/jira",
+	  "email" : "paul.culmsee@sevensigma.com.au"
+    }
+  },
+  "host": "jira.atlassian.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/rest/api/2/search": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "expand": {
+                  "type": "string",
+                  "description": "expand"
+                },
+                "startAt": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "startAt"
+                },
+                "maxResults": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "maxResults"

```

Comment:
@paulculmsee Can you please provide valid description values of the properties as they seem like property Ids.

Documentation - [Description values of properties](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the similar description values in the swagger.

Reply:
Applied to all JIRA's publicly disclosed issue field types as per their documentation. Could not do them all however